revisions/service/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Production deployment spec for revisions announcer service.
+
+# Base golang 1.10 image.
+FROM gcr.io/gcp-runtimes/go1-builder:1.10 as builder
+
+RUN apt-get update
+RUN apt-get install -qy --no-install-suggests git
+WORKDIR /go/src/app
+COPY main.go .
```

Comment:
What about copying the whole directory to make it more future-proof? Otherwise it could be hard to figure out what goes wrong if a new file is added in some distant future.

Reply:
Sure. We don't want/need `Dockerfile` or `app.yaml`, but we could do `COPY *.go .`